components/Media/MediaWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import { cnb } from 'cnbuilder';
+import { AnimateInView, type AnimationType } from '@/components/Animate';
+import { Caption, type CaptionProps } from './Caption';
+import { WidthBox, type WidthType } from '@/components/WidthBox';
+import { type PaddingType } from '@/utilities/datasource';
+import { imageAspectRatios, type ImageAspectRatioType } from '@/utilities/datasource';
+import * as styles from './MediaWrapper.styles';
+
+/**
+ * This is a wrapper component for images and video components.
+ * that provides a shared set of layout and caption options.
+ */
+
+export type MediaWrapperProps = React.HTMLAttributes<HTMLDivElement> & CaptionProps & {
+  aspectRatio?: ImageAspectRatioType;
+  isFullHeight?: boolean;
+  isParallax?: boolean;
+  boundingWidth?: 'site' | 'full';
+  width?: WidthType;
+  pt?: PaddingType;
+  pb?: PaddingType;
+  animation?: AnimationType;
+  delay?: number;
+  aspectRatioClass?: string; // Additional eg, responsive aspect ratio classes
+};
+
+export const MediaWrapper = ({
```

Comment:
This is the wrapper component that's used for StoryImage and StoryVideo that provides it with a common set of options